release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2732,6 +2732,54 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the image registry did not support Amazon Web Services (AWS) region `ca-west-1`. With this release, the image registry can now be deployed in this region. (link:https://issues.redhat.com/browse/OCPBUGS-33672[*OCPBUGS-33672*])
+
+* Previously, ServiceAccounts (SAs) could not be used as OAuth2 clients because there were no tokens associated with the SAs. With this release, the OAuth registry client has been modified to anticipate this case and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33210[*OCPBUGS-33210*])
+
+* Previously, the proxy information set in the install config was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
+
+* Previously, the proxy information set in the install config was not applied to the bootstrap process. With this release, the proxy information is applied to the bootstrap ignition data which is applied to the bootstrap machine and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33205[*OCPBUGS-33205*])
+
+* Previously, the information from `imageRegistryOverrides` was only extracted once on the HyperShift Operator initialization and did not refresh. With this release, the Hypershift Operator retrieves the new `ImageContentSourcePolicy` files from the management cluster and adds them to the Hypershift Operator and Control Plane Operator in every reconciliation loop. (link:https://issues.redhat.com/browse/OCPBUGS-33117[*OCPBUGS-33117*])
+
+* Previously, the Hypershift Operator was not using the `RegistryOverrides` mechanism to inspect the image from the internal registry. With this release, the metadata inspector works as expected during the Hypershift Operator reconciliation, and the `OverrideImages` are properly populated. (link:https://issues.redhat.com/browse/OCPBUGS-32220[*OCPBUGS-32220*])
+
+* Previously, attempting to update the OpenShift vSphere connection configuration failed if the configuration included `””` characters. With this release, the characters are stored correctly and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31863[*OCPBUGS-31863*])
```

Comment:
Wonder if this could more specifically be:
```suggestion
* Previously, attempting to update the {vmw-first} connection configuration for {product-title} failed if the configuration included `””` characters. With this release, the characters are stored correctly and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-31863[*OCPBUGS-31863*])
```